10-inference.Rmd

```diff
@@ -0,0 +1,806 @@
+population_distribution <-  ggplot(airbnb, aes(x = price)) +
+    geom_histogram(fill="#0072B2", color="#e9ecef") +
+    xlab("Price per night ($)") 
+population_distribution
+population_parameters <- airbnb %>% 
+    summarize(pop_mean = mean(price),
+             pop_sd = sd(price))
+population_parameters
+```
+
+We see that the distribution has one peak and is skewed&mdash;most of the listings
+are less than \$250 per night, but a small proportion of listings cost more
+than that, creating a long tail on the histogram's right side. 
+The population mean is \$`r round(population_parameters$pop_mean,2)` and the
+population standard deviation is \$`r round(population_parameters$pop_sd, 2)`.
+
+Suppose we take a sample of 20 observations from our population. Below we 
+create a histogram to visualize the
+distribution of observations in the sample,
+and calculate the mean and standard deviation of our sample. These two numbers
+are **point estimates** for the mean and standard deviation of the full population.
+
+```{r 11-example-means3, echo = TRUE, message = FALSE, warning = FALSE, fig.cap = "Distribution of price per night ($) for sample of 20 Airbnb listings"}
+sample_1 <- airbnb %>% 
+    rep_sample_n(20)
+head(sample_1)
+sample_distribution <- ggplot(sample_1, aes(price)) + 
+    geom_histogram(fill="#0072B2", color="#e9ecef") +
+    xlab("Price per night ($)") 
+sample_distribution
+estimates <- sample_1 %>% 
+    summarize(sample_mean = mean(price),
+             sample_sd = sd(price))
+estimates
+```
+Recall that the population mean 
+was \$`r round(population_parameters$pop_mean,2)` and the population standard deviation
+was \$`r round(population_parameters$pop_sd,2)`. We see that our point
+estimates for the mean and standard deviation 
+are \$`r round(estimates$sample_mean, 2)` and  \$`r round(estimates$sample_sd,2)`,
+respectively. So our estimates were actually quite close to the population parameters: the mean was 
+about `r round(100*abs(estimates$sample_mean - population_parameters$pop_mean)/population_parameters$pop_mean, 1)`% off,
+while the standard deviation was 
+about `r round(100*abs(estimates$sample_sd - population_parameters$pop_sd)/population_parameters$pop_sd, 1)`% off.
+Note that in practice, we usually cannot compute the accuracy of the estimate, since we do not have access to the population
+parameter; if we did, we wouldn't need to estimate it!
+
+Also recall from the previous section that the point estimate can vary; if 
+we took another random sample from the population, then the value of our statistic may change.
+So then did we just get lucky with our point estimate above?
+How much does our estimate vary across different samples of size 20 in this example? Again, since we have access to the population,
+we can take many samples and plot the **sampling distribution** of the point estimates to get a sense
+for this variation. In this case, we'll use 1500 samples of size 20.
```

Comment:
Again, we have fast laptops, let's up the reps a bit.

Reply:
increased to 15000